common/azure-common/src/main/java/com/azure/common/implementation/util/TypeUtil.java

```diff
@@ -6,6 +6,8 @@
 
 package com.azure.common.implementation.util;
 
+import org.graalvm.compiler.core.common.SuppressFBWarnings;
```

Comment:
No - we don't want to have a dependency on GraalVM here. This can be removed (it doesn't even look like it is being used).

Reply:
Removed in another commit. Not pushed yet.